api/sk-serve/standings/service.go

```diff
@@ -0,0 +1,49 @@
+package standings
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/patrickmn/go-cache"
+)
+
+const (
+	standingsURL      = "https://us-central1-sports-kernel.cloudfunctions.net/getStandings"
+	standingsAudience = "https://us-central1-sports-kernel.cloudfunctions.net/getStandings/"
+)
+
+type Service struct {
+	cache *cache.Cache
```

Comment:
Since we have the cache here we should cache the results from the fetch, or remove the cache from the Service struct.